scripts/serverSetup.md

```diff
@@ -0,0 +1,66 @@
+# Getting started with YesWorkflow Web Components
+
+**Please ensure that JDK 10 and Maven 3+ are installed on your machine before continuing and confirm that JDK 10 is your default JAVA_HOME**
+
+## Deploying the web sever
+In the yw_website directory, change directory into the scripts directory. You should see a number of bash scripts:
+```
+deploy_python.sh
+deploy_python3.sh
+remote_deploy_python.sh
+remote_deploy_python3.sh
+workflow_sample.gv
+```
+If you would like to deploy a test server on your localhost, you may run one of these two files:
```

Comment:
we should specify that a user must not have anything running on port 8000 for the following commands to work